css-fonts-5/Overview.bs

```diff
@@ -139,13 +139,38 @@ Issue(5892):
 </dl>
 
 
+<h3 id="font-size-adjust-desc">
+Default font metrics and glyph size scaling: the 'size-adjust' descriptor
+</h3>
+
+<pre class='descdef'>
+Name: size-adjust
+Value: <<percentage>>
+For: @font-face
+Initial: 100%
+</pre>
+
+The 'size-adjust' descriptor defines a scalar by which glyph outlines and metrics from this font
+are multiplied. Negative values are invalid at parse time.
+
+All metrics values of this font, including glyph advance sizes, are resolved as the given percentage
+multiplied by the used font size. When glyphs are visibly drawn, the visual size of the glyphs is
```

Comment:
I realize it's basically just being moved here, but this description is wrong; see #6133. But that could be fixed as a separate PR if you don't want to deal with it now.

Reply:
I'd like to deal with it in a separate PR. Doing code move and modifications at the same time may easily lose track of things. (Removing `normal` is trivial enough that I'd like to do together...)